### PR TITLE
Updating the About Us section of the Geoviews site

### PR DESCRIPTION
**Pull request by kebowen730**
@philippjfr @jlstevens

### CODE REVIEW

File: doc/about.rst [hunk 1]
```diff
@@ -1,10 +1,10 @@
 About Us
 ========
 
-GeoViews is completely `open source <https://github.com/holoviz/geoviews>`_, available under a BSD license freely for both commercial and non-commercial use. GeoViews is part of the `HoloViz <https://holoviz.org>`_ ecosystem and works well with all the HoloViz tools.
+Geoviews is completely open source, available under a BSD license freely for both commercial and non-commercial use. Geoviews was originally developed with the support of `Anaconda Inc. <https://anaconda.com>`_, and is now maintained by Anaconda developers and community contributors.
```
Comment (jbednar) [inline doc/about.rst:4]:
```suggestion
GeoViews is completely open source, available under a BSD license freely for both commercial and non-commercial use. GeoViews was originally developed with the support of `Anaconda Inc. <https://anaconda.com>`_, and is now maintained by Anaconda developers and community contributors.
```

File: doc/about.rst [hunk 2]
```diff
@@ -1,10 +1,10 @@
 About Us
 ========
```
Comment (jbednar) [inline doc/about.rst:2]:
```suggestion
About
=====
```

File: doc/about.rst [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
+Geoviews is completely open source, available under a BSD license freely for both commercial and non-commercial use. Geoviews was originally developed with the support of `Anaconda Inc. <https://anaconda.com>`_, and is now maintained by Anaconda developers and community contributors.
 
-GeoViews was originally developed through a collaboration between `Anaconda Inc. <https://anaconda.com>`_ and the `Met Office
-<http://www.metoffice.gov.uk>`_, and is now maintained by Anaconda developers and community contributors.  
+Geoviews is part of the `HoloViz <https://holoviz.org>`_ family of tools. The `holoviz.org <https://holoviz.org>`_ website shows how to use Geoviews together with other libraries to solve complex problems, with detailed tutorials and examples. You can see a variety of projects using Geoviews at `examples.pyviz.org <https://examples.pyviz.org>`_, and you can compare Geoviews to other available tools at `pyviz.org <https://pyviz.org>`_.
```
Comment (jbednar) [inline doc/about.rst:6]:
```suggestion
GeoViews is part of the `HoloViz <https://holoviz.org>`_ family of tools. The `holoviz.org <https://holoviz.org>`_ website shows how to use GeoViews together with other libraries to solve complex problems, with detailed tutorials and examples. You can see a variety of projects using GeoViews at `examples.pyviz.org <https://examples.pyviz.org>`_, and you can compare GeoViews to other available tools at `pyviz.org <https://pyviz.org>`_.
```

File: doc/about.rst [hunk 4]
```diff
@@ -1,10 +1,10 @@
...
 
-If you are interested in contributing to GeoViews development, you can see our list of `open issues <https://github.com/holoviz/geoviews/issues>`__ that you could help address, including possible larger `wishlist items <https://github.com/holoviz/geoviews/issues?q=is%3Aopen+is%3Aissue+label%3Awishlist>`__.  And if you are lucky enough to be in
-a position to fund our developers to work on documentation, features, or bug fixes, please contact ``sales@anaconda.com``.
+If you have any questions or usage issues visit the `Geoviews Discourse <https://discourse.holoviz.org/c/geoviews/>`_ site.
```
Comment (jbednar) [inline doc/about.rst:8]:
```suggestion
If you have any questions or usage issues visit the `GeoViews Discourse <https://discourse.holoviz.org/c/geoviews/>`_ site. 
```

File: doc/about.rst [hunk 5]
```diff
@@ -1,10 +1,10 @@
...
 
-If you are interested in contributing to GeoViews development, you can see our list of `open issues <https://github.com/holoviz/geoviews/issues>`__ that you could help address, including possible larger `wishlist items <https://github.com/holoviz/geoviews/issues?q=is%3Aopen+is%3Aissue+label%3Awishlist>`__.  And if you are lucky enough to be in
-a position to fund our developers to work on documentation, features, or bug fixes, please contact ``sales@anaconda.com``.
+If you have any questions or usage issues visit the `Geoviews Discourse <https://discourse.holoviz.org/c/geoviews/>`_ site. 
+
+If you like Geoviews and have built something you want to share, tweet a link or screenshot of your latest creation at @HoloViz_org, along with any other library you used (@HoloViews, @Panel_org, @Datashader, @BokehPlots, @Matplotlib, etc.). Thanks!
```
Comment (jbednar) [inline doc/about.rst:10]:
```suggestion
If you like GeoViews and have built something you want to share, tweet a link or screenshot of your latest creation at @HoloViews, along with any other library you used (@HoloViz_org, @Panel_org, @Datashader, @BokehPlots, @Matplotlib, etc.). Thanks!
```

**Comment by kebowen730** [comment]
@jbednar ready for you to review again

**Pull request merged**